### ISSUE TITLE
Displaced tracheal bronchus

### ISSUE THREAD

**Issue by eswietlik** [obophenotype/human-phenotype-ontology#6630]
**Preferred term label:** Displaced tracheal bronchus

**Synonyms**

**Definition (free text, please give PubMed ID)**
Accessory bronchus originating from trachea replacing one of the segmental branches of the anatomically normal upper lobe bronchus. PMID: 11158647, PMID: 19332762

**Parent term (use hpo.jax.org/app)**
Tracheal bronchus (once created)

**Diseases characterized by this term? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**
https://orcid.org/0000-0002-4095-8489

**Comment by pnrobinson** [comment]
added
